turbine/benches/cluster_nodes.rs

```diff
@@ -45,8 +46,12 @@ fn get_retransmit_peers_deterministic(
             0,
             0,
         );
-        let _retransmit_peers =
-            cluster_nodes.get_retransmit_peers(slot_leader, &shred.id(), /*fanout:*/ 200);
+        let _retransmit_peers = cluster_nodes.get_retransmit_addrs(
+            slot_leader,
+            &shred.id(),
+            200, // fanout
```

Comment:
i know this is just a benchmark, but is it worth importing `DATA_PLANE_FANOUT` so this gets updated if fanout ever changes?

Reply:
The problem is that benchmark code cannot access private or even `pub(crate)` symbols.
So we would need to make its definition `pub`, which takes away all the benefits of keeping things private/encapsulated.

anyways, that can be revisited separate from this PR.